client/flutter/divkit/lib/src/generated_sources/div_input.dart

```diff
@@ -1,6 +1,7 @@
 // Generated code. Do not modify.
 
 import 'package:equatable/equatable.dart';
+import 'package:flutter/services.dart';
```

Comment:
Making changes by editing the generated code is unacceptable. After re-generation, when updating the schema, the changes will be rigidly discarded.

It seems that your change does not require any edits to this part.